### ISSUE TITLE
EAP336 open issues

### ISSUE THREAD

**Issue by eu-genia** [SChAth/ScAthiop#2360]
Just to collect here all open questions concerning EAP submissions

Mota Giyorgis https://github.com/BetaMasaheft/Manuscripts/pull/2108 one set not complete
Mamher Wolde Iyasus 001 to 005 no images, Mamher Solomon EAP336_MS005 unclear https://github.com/BetaMasaheft/Manuscripts/pull/2107

https://github.com/BetaMasaheft/Manuscripts/issues/2115

Collections not submitted to BL but to us
"Mamher Kidane Maryam" -  @MershaMengistie from the description it seems that the MSS are now in Manbara Leul Qeddus Marqos (Addis Ababa) church - should this be better used as the current repository, with Abba KM as subcollection?

**Comment by eu-genia** [comment]
MS | TIFF according to BL spreadsheet | Images in the folder
-- | -- | --
Mamher Solomon EAP336_MS005 | 40 | 294
Mamher Solomon EAP336_MS006 | 58 | 54
Mota Giorgis EAP336_MG004 | 357 | 56 - only few images submitted

**Comment by MershaMengistie** [comment]
All are from Dima Giyorgis...but 009 is not an andemta text but the biography of one of the former instructors of the monastery. I don't think it fits into the project and should not be posted.

**Comment by eu-genia** [comment]
For BM all manuscripts fit. Or we can use some as additional Abennat material. so I would create stubs for all, just need the repositories.

**Comment by eu-genia** [comment]
The mismatches in EAP432 images also still to be fixed

**Comment by eu-genia** [comment]
repository/collection for C4-IV-123 (formerly submitted as PDF under EAP - Mamher Demissie - 002 but no match among the TIFFs in the EAP336 image folders) not identified

**Comment by eu-genia** [comment]
EAP must also be mapped to EMDA, the same manuscripts were also submitted to HMML.
Mota Giyorgis

 EAP336/3/1 = EMDA 72
 EAP336/3/2 = EMDA 73
 EAP 336/3/3 = EMDA 74
 EAP336/3/5 = EMDA 78
 EAP 336/3/6 = EMDA 84
 EAP 336/3/7 = EMDA 86
 EAP 336/3/8 = EMDA 88
 EAP 336/3/9 = EMDA 90

Martula Maryam ? etc

**Comment by MershaMengistie** [comment]
Yes, those sites were visited for both projects I worked on for BL and HMML.

**Comment by eu-genia** [comment]
This is a huge problem, as now the same manuscripts are available online on three different resources, with different shelfmarks, this may never happen. It seems that HMML does not have the information about the EAP shelfmarks and the other way around.

**Comment by eu-genia** [comment]
Digitizing twice of course makes no sense and would not anyhow improve the situation - 
but unique shelfmarks and mapping (and preferably not submitting the same results independently) are essential.

**Comment by MershaMengistie** [comment]
That is true. I did that with the knowledge of all these.

**Comment by eu-genia** [comment]
@MershaMengistie you don't by any chance have a list of EMDA mss with shelfmarks/numbers?

**Comment by MershaMengistie** [comment]
No I don't have, dear Zhenia.

**Comment by eu-genia** [comment]
EAP 432/1/53 submitted for EAP 432 as Qaranyo is actually from Mota = EMDA 104!
